src/common/portal-metadata-context/PortalMetadataContext.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { IContextItem, IPortalMetadata } from "./Interfaces";
+
+export class PortalMetadataContext {
+
+    private portalMetadata: IPortalMetadata;
+    private portalMetadataContext: IContextItem;
+
+
+    public getPortalMetadataContext() {
```

Comment:
lets have a set funciton for initilization here - also this should be a singleton class  -check WebExtensionContext for referecen